src/components/TopBar.js

```diff
@@ -341,29 +344,40 @@ export default _.flow(
               style: { marginLeft: '0.5rem' }
             })
           ]),
-          isFirecloud() && h(Clickable, {
-            style: styles.nav.supportItem,
-            as: 'a',
-            hover: { backgroundColor: colors.gray[3] },
-            href: 'https://broadinstitute.zendesk.com/hc/en-us/articles/360022694271-Side-by-side-comparison-with-Terra',
-            target: '_blank',
-            onClick: () => this.hideNav()
-          }, [
-            div({ style: styles.nav.icon }, [
-              icon('help-info', {
-                className: 'is-solid',
-                size: 20
-              })
-            ]),
-            div([
-              'What\'s different in',
+          isFirecloud() && h(Fragment, [
+            h(Clickable, {
+              style: styles.nav.supportItem,
+              as: 'a',
+              hover: { backgroundColor: colors.gray[3] },
+              href: 'https://broadinstitute.zendesk.com/hc/en-us/articles/360022694271-Side-by-side-comparison-with-Terra',
+              target: '_blank',
+              onClick: () => this.hideNav()
+            }, [
+              div({ style: styles.nav.icon }, [
+                icon('help-info', { className: 'is-solid', size: 20 })
+              ]),
               div([
-                'Terra?',
-                icon('pop-out', {
-                  size: 12,
-                  style: { marginLeft: '0.5rem', flexGrow: 1 }
-                })
+                'What\'s different in',
+                div([
+                  'Terra?',
+                  icon('pop-out', {
+                    size: 12,
+                    style: { marginLeft: '0.5rem', flexGrow: 1 }
+                  })
+                ])
               ])
+            ]),
+            h(Clickable, {
+              style: styles.nav.supportItem,
+              disabled: !isSignedIn,
+              tooltip: isSignedIn ? undefined : 'Please sign in',
```

Comment:
When a user is redirected from portal.firecloud to firecloud.terra, whether they are signed in or not, they see the popup message welcoming them to Terra. In that message, it says that if they want to return to FC then they should go to the Left nav to click "Use Classic FireCloud", so I think it should still show up. There might be a better UX but this seemed like the most straightforward implementation.

Reply:
Yeah, the rationale behind having the tooltip is I think it could be confusing if there's a message directing users to the left nav, and then not having anything there because the user is signed out (and it not being clear that's why).